source/administration/mmctl-cli-tool.rst

```diff
@@ -4170,6 +4171,41 @@ mmctl user email
    --insecure-sha1-intermediate  allows the use of insecure TLS protocols, such as SHA-1
    --local                       allows communicating with the server through a unix socket
    --strict                      will only run commands if the mmctl version matches the server one
+   
+mmctl user verify
+^^^^^^^^^^^^^^^^^
+
+**Description**
+
+    Verify the email address of a new user.
+
+**Format**
+
+.. code-block:: none
+
+   mmctl user verify [users] [flags]
+
+**Example**
+    
+.. code-block:: none
+
+   mmctl user verify user1
+     
+**Options**
+
+.. code-block:: sh
+
+    -h, --help       help for email
+
+**Options inherited from parent commands**
+
+.. code-block:: sh
+
+   --format string               the format of the command output [plain, json] (default "plain")
```

Comment:
Based on the [mmctl repository](https://github.com/mattermost/mmctl/blob/master/docs/mmctl_user_verify.rst), there are 6 inherited options from parent commands:

--config-path string           path to the configuration directory. If "$HOME/.mmctl" exists it will take precedence over the default value (default "$XDG_CONFIG_HOME")
--format string                the format of the command output [plain, json] (default "plain")
--insecure-sha1-intermediate   allows to use insecure TLS protocols, such as SHA-1
--insecure-tls-version         allows to use TLS versions 1.0 and 1.1
--local                        allows communicating with the server through a unix socket
--strict                       will only run commands if the mmctl version matches the server one

Can you update this list to match above, please? Thanks so much!

Reply:
I noticed in the other commands we did not list all the options that were included, so I copied that. I can push the update for this one. Should we also update all the other commands to match?

Reply:
Just updated the mmctl user verify command

Reply:
Great question! Many of the commands in the product documentation are currently outdated when compared against the `mmctl` repository -- and this is one of the most common areas of the docs that need updates. I'm in the process of reviewing all documented commands against the `mmctl` repo and updating accordingly.